### PR TITLE
Logs aggregation solution

### PR DESCRIPTION
**Pull request by Jodao**
Closes issue #1

Added a logs agreggation solution and an explanation on how it was done

### CODE REVIEW

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogManager.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import eubr.atmosphere.tma.database.DatabaseManager;
+import java.sql.Types;
+
+public class LogManager {
+    
+    private static final Logger LOGGER = LoggerFactory.getLogger(LogManager.class);
+
+    
+    public int insertLog(String origin, int originId, String description, String previousValue, String newValue, String component, int logGroupId, String target, int targetId) {
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogManager.java:20]: @Jodao , I think this method needs to be static. Is there any reason for not being static?

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogManager.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            
+            ps.setString(9,target);
+            
+            //check if targetId is not null
+            if(targetId == -1)
+                ps.setNull(10, Types.INTEGER);
+            else
+                ps.setInt(10,targetId);
+            
+            DatabaseManager databaseManager = new DatabaseManager();
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogManager.java:61]: @Jodao , I think you should move this line to before the "creation of the preparedStatement". Then, you should use this object to getConnectionInstance() ...

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+This is a component from TMA-Utils that will allow each component of the TMA to save its logs on the database. For that, each component must import the package TMA-Utils and make use of the class "LogManager"(i.e. "import eubr.atmosphere.tma.logs.LogManager").
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/README.md:1]: space before "(".

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Whenever we want to log on the database we call the only method "insertLog" of LogManager's Class and give it the parameters according to what we are logging.
+
+For saving the data on the database, this class invokes another class, DatabaseManager, present in the package `eubr.atmosphere.tma.database`.
+
+
+Next, we present the table that represent this logs on the database and we describe each parameter of it.
+
+![Log table used to save logs on database](https://github.com/Jodao/TMA/blob/main/tma-framework/common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogsTable.png)
+
+Fields explanation:
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/README.md:12]: Try to be consistent here. Try to explain in the same format for each item. 
Also, follow a pattern. Either add a "." at the end of each item or do not add to any of them.

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogManager.java (hunk 1 above)
Comment (Jodao) [reply common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogManager.java]: No reason. Fixed it.

File: common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/README.md [hunk 5]
```diff
@@ -7,7 +7,7 @@ For saving the data on the database, this class invokes another class, DatabaseM
 
 Next, we present the table that represent this logs on the database and we describe each parameter of it.
 
-![Log table used to save logs on database](https://github.com/Jodao/TMA/blob/main/tma-framework/common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogsTable.png)
+![Log table used to save logs on database](https://github.com/Jodao/tma-framework/blob/feat/logs_aggregation/common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/LogsTable.png)
```
Comment (joseadp) [inline common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/README.md:10]: @Jodao , please, change the path to a relative path.

Comment (Jodao) [reply common/tma-utils/src/main/java/eubr/atmosphere/tma/logs/README.md]: Done

**Review by joseadp — APPROVED** [review]
LGTM

**Review by nmsa — APPROVED** [review]

**Pull request merged**